tests/main/lxd-no-fuse/task.yaml

```diff
@@ -1,4 +1,4 @@
-summary: Check that we can install snaps when fuse is missing in lxd
+summary: Check that we can install snaps when fuse/fuse3 is missing in lxd
```

Comment:
I'm afraid the static checker only complains about changed files. Can you add a details section? Say something like:
```
details: |
    Verify that fuse/fuse3 is pulled in as a dependency when installing snapd from deb.
```